v2/spanner-common/src/main/java/com/google/cloud/teleport/v2/spanner/ddl/InformationSchemaScanner.java

```diff
@@ -145,9 +145,12 @@ private void listTables(Ddl.Builder builder) {
       String tableName = resultSet.getString(0);
       String parentTableName = resultSet.isNull(1) ? null : resultSet.getString(1);
       String onDeleteAction = resultSet.isNull(2) ? null : resultSet.getString(2);
+      String interleaveType = resultSet.isNull(3) ? null : resultSet.getString(3);
 
       // Error out when the parent table or on delete action are set incorrectly.
-      if (Strings.isNullOrEmpty(parentTableName) != Strings.isNullOrEmpty(onDeleteAction)) {
+      if (interleaveType != null
```

Comment:
this class is used in the import export flow as well. We can not assume only 1 type of interleaving here.

Reply:
Again as stated above, the code flow will continue to be the same for INTERLEAVE IN PARENT